charts/trillian/templates/trillian-log-server/deployment.yaml

```diff
@@ -68,6 +68,7 @@ spec:
           args:
 {{  include "trillian.logServer.args" . | indent 12 }}
           env:
```

Comment:
since logServer and logSigner have the same `env` args, does it make sense to hoist it into _helpers.tpl? to avoid duplication.

Reply:
sure, I can do that.